SparkPost/2web2ui
# SA-1995: UX Feedback for Blocklist Incident Details Page

Note: Give your PR a short title. Something like "_TICKET NUMBER: feature description_" is good.

### What Changed
- Removed header for remediation steps.
- Changed resource to either show IP or Domain
- Changed link text to be lowercase.
- Updated empty state for tables
- Added sort for table headers
- Moved icon to right align.

### How To Test

- Point local dev to production
- Go to `/signals/blocklist/incidents`
- Click on any incident.
- Verify that the above works. (Will need to check IP incident, Domain incident, and incident with empty related incidents.)
